src/clickonce/shared/NativeMethods.cs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Deployment.Utilities
+{
+    internal static class NativeMethods
+    {
+        // Warning - Launcher resource type and name should never be changed
+        public static readonly IntPtr Launcher_CustomResourceTypePtr = new IntPtr(50);
+        public const string Launcher_ResourceName = "FILENAME";
+
+        public const UInt32 LOAD_LIBRARY_AS_DATAFILE = 0x00000002;
+
+        [DllImport("kernel32.dll", CharSet = CharSet.Unicode, SetLastError = true)]
+        internal static extern IntPtr BeginUpdateResourceW(String fileName, bool deleteExistingResource);
```

Comment:
The class is internal so I typically recommend changing the members to `pubic`.

Reply:
Thanks - I''ll fix it.